fuzzers/libfuzzer_libpng_norestart/src/lib.rs

```diff
@@ -11,7 +11,7 @@ use std::{env, net::SocketAddr, path::PathBuf};
 
 use clap::{self, Parser};
 use libafl::{
-    corpus::{Corpus, InMemoryOnDiskCorpus, OnDiskCorpus},
+    corpus::{Corpus, InMemoryCorpus, OnDiskCorpus},
```

Comment:
Why change the corpus type here?

Reply:
Whoops, that was not meant to be committed. I was using it for testing.